docker/gitlab-content-ci/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM demisto/python3-deb:3.9.1.14969
+
+RUN apt-get update
+
+RUN apt-get install -y curl jq unzip openssh-client gcc libffi-dev libssl-dev git zip netcat wget python2 python-pip
+
+# Currently sh is linked to dash, linking it to /bin/bash instead
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+# Installing nvm, npm and node.js
+ENV NODE_VERSION 12.18.4
+RUN \
+    curl https://raw.githubusercontent.com/creationix/nvm/v0.38.0/install.sh | bash \
```

Comment:
Why is this,
`https://raw.githubusercontent.com/creationix/nvm/v0.38.0/install.sh`
and not,
`https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh`
?

Reply:
good catch